.github/pull_request_template.md

```diff
@@ -1,3 +1,8 @@
+<!-- If this is deployment-related, please go the the Preview tab and select the appropriate sub-template. Otherwise, delete everything before #Description -->
```

Comment:
```suggestion
<!-- If this is deployment-related, please go the the Preview tab and select the appropriate sub-template. -->
<!-- Otherwise, delete everything before #Description -->
```
Seeing it live, I think it reads better with Otherwise on a new line.